_posts/2016/2016-08-04-dev-onboarding.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "5 steps to make you feel comfortable in your new job"
+date: "2016-08-04 07:00:28"
+image: blog-header/onboarding.jpg
+categories: working-at-epages
+authors: ["Birgit, Leana", "Serena"]
+---
+
+You've spent days and weeks perfecting your resume, preparing for interviews, and making a good impression at personal meetings.
+Now that the application process is over, and you've landed the job, it's definitely time to throw confetti.
+But when the first working day at the new company approaches, you wonder what your first days, weeks, and months will be like.
+Being nicely welcomed and appreciated ensures that we keep our first working day in good memory.
```

Comment:
that we keep our first working day in good memory.
->
that we fondly remember our first working day.